tests/unit/steps/addNewSlides.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { ConfigService } from '@nestjs/config';
+import { Content } from '../../../src/confluence/confluence.interface';
+import { ContextService } from '../../../src/context/context.service';
+import addNewSlides from '../../../src/proxy-page/steps/addNewSlides';
+import { createModuleRefForStep } from './utils';
+
+describe('ConfluenceProxy / addNewSlides', () => {
+    let context: ContextService;
+    let config: ConfigService;
+    let content: Content;
+
+    beforeEach(async () => {
+        const moduleRef = await createModuleRefForStep();
+        context = moduleRef.get<ContextService>(ContextService);
+        config = moduleRef.get<ConfigService>(ConfigService);
+    });
+
```

Comment:
I guess we are missing some testing cases:

- Add slides
- Add slides with vertical slides
- Display slides over legacy slides via page properties
- Add slides with background image
- Add slides with background video
- Add slides with fragments
- Add slides with special styles (cover, default and blank)
- Add slides with different transitions